manifests/crds/cpet.belastingdienst.nl_paasconfig.yaml

```diff
@@ -241,9 +231,7 @@ spec:
                 - namespace
                 type: object
             required:
-            - capabilities
             - decryptKeyPaths
-            - exclude_appset_name
```

Comment:
In the current implementation, we would get weird behaviour making this optional, as the following code will add an empty name to the ignored object: https://github.com/belastingdienst/opr-paas/blob/main/internal/controller/argo_app.go#L99

Will be removed from code once ArgoCD specific code has been extracted to a capability. Therefore I would like to make this required until then.